frontend/src/pages/Browse/Browse.tsx

```diff
@@ -1,11 +1,30 @@
-import { Card } from "@/components/Card";
+import { ProductColumn } from "@/components/ProductColumn";
 import { Input } from "@/components/retroui";
 
 import nwData from "../../../../backend/app/assets/data/nw_actual.json";
 import pnsData from "../../../../backend/app/assets/data/pns_actual.json";
 import wlsData from "../../../../backend/app/assets/data/wls.json";
 
 export function Browse() {
+  if (!nwData && !pnsData && !wlsData) {
+    return (
+      <div>
+        <div className="px-24 flex justify-center border-b border-b-black">
+          <div className="w-[90%] my-14  ">
+            <Input type="text" placeholder="Search Product" />
+          </div>
+        </div>
+        <p>Something has gone wrong, try again...</p>
+      </div>
+    );
+  }
+
```

Comment:
Use fallback JSON only if props are undefined

```suggestion
  const nw = nwProp ?? nwData;
  const pns = pnsProp ?? pnsData.results;
  const wls = wlsProp ?? wlsData;
```